packages/gatsby-theme-docs/src/layouts/content-homepage.js

```diff
@@ -23,7 +23,6 @@ const LayoutContentHomepage = (props) => {
         {...layoutState.sidebar}
         {...layoutState.searchDialog}
         siteTitle={siteData.siteMetadata.title}
-        isGlobalBeta={props.pageData.isGlobalBeta}
```

Comment:
The global beta flag is a different thing and should stay.

<img width="227" alt="image" src="https://user-images.githubusercontent.com/1110551/95593762-61d2e180-0a4a-11eb-9bad-79e0fa0c427f.png">

Only the "chapter" page beta flags should be moved to the page index nav.

Reply:
Wasn't sure about that...thanks 👍

Reply:
For clarification:  "Page" beta, not Chapter.